Workbooks/UsageMetrics/Usage - Events/Usage - Events.workbook

```diff
@@ -756,6 +756,7 @@
         "query": "{barChartQuery}",
         "size": 0,
         "aggregation": 3,
+        "showAnnotations": true,
```

Comment:
For my knowledge, what does this parameter do?

Reply:
If the visualization shown has time as its x axis, setting this will show annotation markers at the top of the chart.